llvm/test/ThinLTO/X86/workload.ll

```diff
@@ -1,67 +1,153 @@
+; Set up
```

Comment:
Nit: add brief comment at the top about what this is testing (e.g. the workload based importing via -thinlto-workload-def.

Also suggest moving the current m1.ll, m2.ll, and m3.ll code to the end so that all the test RUN lines and checks are at the top.

Reply:
Done.